Firebase/Auth/Source/AuthProviders/Phone/FIRPhoneAuthProvider.m

```diff
@@ -201,6 +371,66 @@ - (void)verifyClientWithCompletion:(FIRVerifyClientCallback)completion {
   }];
 }
 
+- (void)reCAPTCHAURLWithCompletion:(FIRReCAPTCHAURLCallBack)completion {
+  [self fetchAuthDomainWithCompletion:^(NSString *_Nullable authDomain,
+                                        NSError *_Nullable error) {
+    if (error) {
+      completion(nil, error);
+      return;
+    }
+    NSString *bundleID = [NSBundle mainBundle].bundleIdentifier;
+    NSString *clienID = _auth.app.options.clientID;
+    NSString *apiKey = _auth.requestConfiguration.APIKey;
+    NSMutableDictionary *urlArguments = [[NSMutableDictionary alloc] initWithDictionary: @{
+      @"apiKey" : apiKey,
+      @"authType" : kAuthTypeVerifyApp,
+      @"ibi" : bundleID,
+      @"clientId" : clienID,
+      @"v" : [NSString stringWithFormat:@"FirebaseAuth.iOS/%s %@",
```

Comment:
Please use a common method for this. Later when we're going to change it both need be changed.

Reply:
Done, added as a class method to FIRAuthBackend.